### PR TITLE
Added Increase Command test cases

### PR DESCRIPTION
**Pull request by Bisceto**
Added test cases for IncreaseCommand and IncreaseCommandParser

### CODE REVIEW

File: src/test/java/seedu/wife/testutil/IncreaseFoodDescriptorBuilder.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package seedu.wife.testutil;
+
+import seedu.wife.logic.commands.foodcommands.IncreaseCommand.IncreaseFoodDescriptor;
+import seedu.wife.model.food.Food;
+import seedu.wife.model.food.Quantity;
+
+/**
+ * A utility class to help with building IncreasePersonDescriptor objects.
```
Comment (Jnjy) [inline src/test/java/seedu/wife/testutil/IncreaseFoodDescriptorBuilder.java:8]: Can you change this to `IncreaseFoodDescriptor`? Same thing for line 31 as well

**Review by Jnjy — CHANGES_REQUESTED** [review]

File: src/test/java/seedu/wife/testutil/IncreaseFoodDescriptorBuilder.java (hunk 1 above)
Comment (Bisceto) [reply src/test/java/seedu/wife/testutil/IncreaseFoodDescriptorBuilder.java]: resolved

Comment (Bisceto) [reply src/test/java/seedu/wife/testutil/IncreaseFoodDescriptorBuilder.java]: resolved

**Review by Jnjy — APPROVED** [review]
LGTM

**Pull request merged**